samples/notebooks/image-captioning-gcp/README.md

```diff
@@ -0,0 +1,40 @@
+# Image Captioning TF 2.0
+
+## About
```

Comment:
How about "Overview"? We try to make the readme more consistent across all samples.

Reply:
About --> Overview